music21/repeat.py

```diff
@@ -4376,8 +4376,9 @@ def test_expand_repeats(self):
         p = converter.parse('tinyNotation: c1 d e f')
         p.measure(2).storeAtEnd(bar.Repeat(direction='end', times=1))
         p.partName = 'mypartname'
+        p.partAbbreviation = 'mypartabbreviation'
         exp = p.expandRepeats()
-        self.assertEqual(exp.partName, 'mypartname')
+        self.assertEqual(exp.partAbbreviation, 'mypartabbreviation')
```

Comment:
Bear with me while I make this changes in the GitHub.com interface :D
```suggestion
        self.assertEqual(exp.partName, 'mypartname')
        self.assertEqual(exp.partAbbreviation, 'mypartabbreviation')
```